high_card.rb

```diff
@@ -1,5 +1,15 @@
 # Use these two arrays to generate a deck of cards.
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
+deck = []
+
+  ranks.each_with_index do |r, i|
+    suits.each do |suit|
+      deck.push({ ranks: r, suits: suit, })
+    end
+  end
+end
+
+  deck.shuffle
 
 players = []
```

Comment:
I understand the need to timebox but try pseudo-coding the next part so it would make it easier to come back to...
```
# Collect an array of players:
# Prompt user for names
# Add names to players array
# Repeat until user enters 'play'

# Deal each player a card


```